scripts/gcp_setup/bash_scripts/run_forseti.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /home/ubuntu/forseti_env.sh
+
+# Put the config files in place.
+gsutil cp gs://$SCANNER_BUCKET/configs/forseti_conf.yaml $FORSETI_CONF
+gsutil cp -r gs://$SCANNER_BUCKET/rules $FORSETI_HOME/
+
+if [ ! -f "$FORSETI_CONF" ]; then
+    echo "Forseti conf not found, exiting."
+    exit 1
+fi
+
+# inventory command
+forseti inventory create
```

Comment:
So another option here would be to create a model ID based on timestamp and then do:
MODELID=$(/bin/date -u +%Y%m%dT%H%M%S)
forseti inventory create --import_as ${MODEL_ID}
forseti model use ${MODEL_ID}

...
scanner stuff
...
Something to clean up inventory ids older than X days and to clean up models older than Y days?

Reply:
This is even better, thanks!

Do you think customers want to configure the cleanup of inventory/models?